### DISCUSSION TITLE
integrazione ALEXA

### DISCUSSION THREAD

**Discussion by Virgilio123** [indomus/forum#1214] [GENERALE - Discussioni generiche]
Buongiorno, ho seguito le istruzioni per creare l'integrazione con ALEXA.

quando vado a fare il test ricevi questo messaggio:

```
Test Event Name
test

Response
{
  "errorMessage": "Unable to import module 'lambda_function'"
}

Function Logs
START RequestId: 3db2fce9-af33-4b32-a010-6802c34ceffb Version: $LATEST
Unable to import module 'lambda_function': No module named 'lambda_function'

END RequestId: 3db2fce9-af33-4b32-a010-6802c34ceffb
REPORT RequestId: 3db2fce9-af33-4b32-a010-6802c34ceffb	Duration: 1.32 ms	Billed Duration: 2 ms	Memory Size: 128 MB	Max Memory Used: 42 MB

Request ID
3db2fce9-af33-4b32-a010-6802c34ceff
```

non riesco a capire il perché, mi sembra dio aver fatto tutti i passaggi correttamente.

Grazie per l'aiuto.

**Comment by indomus** [comment]
E' probabile tu abbia pasticciato qualcosa, specialmente il passo relativo al download del pacchetto .zip.
Ribatti tutto con molta attenzione.
